cmd/operator-sdk/main.go

```diff
@@ -33,7 +33,7 @@ func main() {
 	// a Kubebuilder project as a way to opt into the new KB CLI.
 	// TODO: Make the new KB CLI the default, once the integration is complete
 	// and deprecate "operator-sdk new" from the old CLI.
-	if kbutil.IsConfigExist() {
+	if kbutil.HasProjectFile() {
```

Comment:
Keep compatible with the name that will be used soon in kubebuilder for it.

Reply:
This seems out of scope for this PR. Can we make a separate PR so we can discuss each change on its merits separately?